man/mkinitcpio.8.txt

```diff
@@ -139,6 +139,14 @@ Options for UEFI executable
 	Include a os-release file for the UEFI executable.
 	Default: '/etc/os-release' or '/usr/lib/os-release'.
 
+Options for preset
```

Comment:
Remove this section seperation and mention `--remove` inline. Specify it can only be used with `--preset`.